docs/README.md

```diff
@@ -1,43 +1,78 @@
 # Block Editor Handbook
 
-**Gutenberg** is a codename for a whole new paradigm in WordPress site building and publishing, that aims to revolutionize the entire publishing experience as much as Gutenberg did the printed word. Right now, the project is in the second phase of a four-phase process that will touch every piece of WordPress -- Editing, **Customization** (which includes Full Site Editing, Block Patterns, Block Directory and Block based themes), Collaboration, and Multilingual -- and is focused on a new editing experience, the block editor (which is the topic of the current documentation).
+Hi! 👋 Welcome to the Block Editor Handbook.
+
+The **block editor** is a modern and up-to-date paradigm for WordPress site building and publishing.
+
+It uses a modular system of **Blocks** to compose and format content, and is designed to create rich and flexible layouts for websites and digital products.
+
+The editor consists of a number of key elements which are highlighted in the following graphic:
 
 ![Quick view of the block editor](https://raw.githubusercontent.com/WordPress/gutenberg/trunk/docs/assets/quick-view-of-the-block-editor.png)
 
 **Legend:**
 
-1. Block inserter
-2. Block editor content area
-3. Settings sidebar
+1. _Block inserter_
+2. _Block editor content area_
+3. _Settings sidebar_
```

Comment:
This should probably be called `Sidebar Inspector` or `Settings Sidebar`. It also contains page settings and global styles, not just block settings.